expression/aggregation/sum.go

```diff
@@ -44,13 +42,6 @@ func (sf *sumFunction) Update(ctx *AggEvaluateContext, sc *stmtctx.StatementCont
 
 // GetResult implements Aggregation interface.
 func (sf *sumFunction) GetResult(ctx *AggEvaluateContext) (d types.Datum) {
-	if ctx.Value.Kind() == types.KindFloat64 {
```

Comment:
why remove this?

Reply:
@XuHuaiyu There should not be this branch. If `ctx.Value.Kind() == types.KindFloat64`, the return type should also be `Double` other than `NEWDECIMAL`. so just return `ctx.Value` is ok